modules/data/document/contrib/generation/src/EventSubscriber/DocumentPDFGenerationSubscriber.php

```diff
@@ -73,10 +73,10 @@ public function generateDocumentPDF(WorkflowTransitionEvent $event) {
       'pdf_style' => $document->pdf_style->entity,
       '__destination' => $field_def->getSetting('uri_scheme', 'private'). '://'
         . $field_def->getSetting('file_directory', 'documents')
-        . '/' . preg_replace('[^a-z0-9_-]+', '-', $document->label->value) . '.pdf',
+        . '/' . preg_replace('/[^A-z0-9_]+/', '-', $document->label->value) . '.pdf',
```

Comment:
You would normally write this as '/[A-Za-z0-9]+/', or better still (if its that you don't care about the case) '/[a-z0-9]+/i'

The i makes it case **i**nsensitve

Reply:
Done.